pages/design/composed-template-empty.md

```diff
@@ -0,0 +1,78 @@
+---
+name:
+description: |
+  This rule checks ...
+
+success_criterion:
+- x.y.z
+
+atomic_rules:
+- 
+
+authors:
+-
+---
+
+## Aggregation Definition
```

Comment:
Apparently the word "Aggregation" may be replaced with something else that references how a set of atomic rules work together for a composed ruled. Maybe "Composition Definition" or "Composed Definition".